charts/cht-user-management/Chart.yaml

```diff
@@ -12,7 +12,18 @@ dependencies:
   - alias: cht-user-management
     name: base-single-app
     version: "0.2.0"
+    #repository: file://../base-single-app
     repository: https://docs.communityhealthtoolkit.org/helm-charts
-    # repository: file://../base-single-app
+
+  - alias: cht-user-management-worker
+    name: base-single-app
+    version: "0.2.0"
+    repository: https://docs.communityhealthtoolkit.org/helm-charts
+    #repository: file://../base-single-app
+
+  - alias: redis
+    name: redis
+    version: "16.13.2"
```

Comment:
How does Bitnami's `16.13.2` [version](https://github.com/bitnami/charts/releases/tag/redis%2F16.13.2) map to the Redis version [we're using in Docker](https://github.com/medic/cht-user-management/blob/496066f956a6935f7a6e7f1dac4d723da9608a19/docker-compose.yml#L3C5-L3C28),  `6.2` on Alpine?  

I spent 10 minutes looking and didn't see an obvious way to tell.

I did find the official helm repo that was deprecated and [said  to use bitnami](https://github.com/helm/charts/blob/master/stable/redis/README.md#this-helm-chart-is-deprecated) - so TIL!

Reply:
> How does Bitnami's 16.13.2 [version](https://github.com/bitnami/charts/releases/tag/redis%2F16.13.2) map to the Redis version [we're using in Docker](https://github.com/medic/cht-user-management/blob/496066f956a6935f7a6e7f1dac4d723da9608a19/docker-compose.yml#L3C5-L3C28), 6.2 on Alpine?

[Changelong](https://github.com/bitnami/charts/tree/main/bitnami/redis#to-1700) references version 17.0 as `This major version updates the Redis® docker image version used from 6.2 to 7.0, the new stable version`, so I used the latest 16.13.2 which contains redis 6.2